holoviews/plotting/mpl/chart.py

```diff
@@ -679,104 +616,73 @@ def _get_map_info(self, vmap):
         """
         Get the minimum sample distance and maximum magnitude
         """
-        dists = []
-        for vfield in vmap:
-            dists.append(self._get_min_dist(vfield))
-        return min(dists)
-
-
-    def _get_info(self, vfield, input_scale, ranges):
-        ndims = len(vfield.dimensions())
-        xs = vfield.dimension_values(0) if len(vfield.data) else []
-        ys = vfield.dimension_values(1) if len(vfield.data) else []
-        radians = vfield.dimension_values(2) if len(vfield.data) else []
-        magnitudes = vfield.dimension_values(3) if ndims>=4 else np.array([1.0] * len(xs))
-        colors = magnitudes if self.color_dim == 'magnitude' else radians
-
-        if ndims >= 4:
-            magnitude_dim = vfield.get_dimension(3).name
-            _, max_magnitude = ranges[magnitude_dim]
-        else:
-            max_magnitude = 1.0
-
-        min_dist =      self._min_dist if self._min_dist else self._get_min_dist(vfield)
-
-        if self.normalize_lengths and max_magnitude != 0:
-            magnitudes =  magnitudes / max_magnitude
-
-        return (xs, ys, list((radians / np.pi) * 180),
-                magnitudes, colors, input_scale / min_dist)
+        return np.min([self._get_min_dist(vfield) for vfield in vmap])
 
 
     def _get_min_dist(self, vfield):
         "Get the minimum sampling distance."
-        xys = np.array([complex(x,y) for x,y in zip(vfield.dimension_values(0),
-                                                    vfield.dimension_values(1))])
+        xys = vfield.array([0, 1]).view(dtype=np.complex128)
```

Comment:
Good point, it's relying on the two columns being 64-bit, will have to double check this or revert it.